tee-worker/service/src/main.rs

```diff
@@ -162,17 +163,22 @@ fn main() {
 
 	let clean_reset = matches.is_present("clean-reset");
 	if clean_reset {
-		setup::purge_files_from_cwd().unwrap();
+		#[allow(clippy::unwrap_used)]
+		{
+			setup::purge_files_from_cwd().unwrap();
+		}
 	}
 
 	// build the entire dependency tree
 	let tokio_handle = Arc::new(GlobalTokioHandle {});
+	#[allow(clippy::unwrap_used)]
 	let sidechain_blockstorage = Arc::new(
 		SidechainStorageLock::<SignedSidechainBlock>::new(PathBuf::from(&SIDECHAIN_STORAGE_PATH))
 			.unwrap(),
 	);
 	let node_api_factory =
 		Arc::new(NodeApiFactory::new(config.node_url(), AccountKeyring::Alice.pair()));
+	#[allow(clippy::unwrap_used)]
```

Comment:
Do you have any good idea to remove this unwrap? Actually it frequently causes trouble.

Reply:
Rust noob here, but if the `enclave_init` (or similar critical initialization) fails then shouldn't the error be bubbled up with `?` :thinking:?